Makefile

```diff
@@ -55,7 +55,7 @@ lint: bin/golint
 
 .PHONY: docker-image
 docker-image:
-	@sudo docker build -t $(DOCKER_IMAGE) .
```

Comment:
can you revert this change? docker requires (or should require) sudo on linux

Reply:
@ericchiang -- Good point. Reverted!